jlm/llvm/opt/RvsdgTreePrinter.hpp

```diff
@@ -77,6 +115,12 @@ class RvsdgTreePrinter final : public optimization
   run(RvsdgModule & rvsdgModule);
 
 private:
+  [[nodiscard]] util::AnnotationMap
+  ComputeAnnotationMap(const rvsdg::graph & rvsdg) const;
```

Comment:
I get it that private methods doesn't contribute to the public API, but wouldn't it still be helpful with documentation for these?

Reply:
Added documentation.